recipe/meta.yaml

```diff
@@ -1,19 +1,17 @@
 {% set name = "optree" %}
-{% set version = "0.12.1" %}
-{% set sha256 = "76a2240e7482355966a73c6c701e3d1f148420a77849c78d175d3b08bf06ff36" %}
+{% set version = "0.14.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: c011c6124d6dcbfceade2d7c4f836eab66ed8cf9ab12f94535b41a71dd734637
 
 build:
-  skip: true  # [python_impl == 'pypy']
   script:
-    - set CMAKE_GENERATOR=Ninja # [win]
+    - set CMAKE_GENERATOR=Visual Studio 16 2019 # [win]
```

Comment:
I think that if you unset `CMAKE_GENERATOR_PLATFORM`, it will work. See https://github.com/metaopt/optree/blob/v0.14.1/setup.py#L101-L108. We should really try to use Ninja where possible since it is much faster on Windows than the Visual Studio generator.

Reply:
If I unset CMAKE_GENERATOR_PLATFORM or CMAKE_GENERATOR like conda-forge did, I get an error:
```
CMake Error: CMAKE_GENERATOR was set but the specified generator doesn't exist. Using CMake default.
  -- Building for: Visual Studio 17 2022
```

It only succeeds if I set   
```
script_env:             # [win]
    - CMAKE_GENERATOR=Visual Studio 16 2019 # [win]
```

Reply:
Have you tried unsetting `CMAKE_GENERATOR_PLATFORM` and setting `CMAKE_GENERATOR` to Ninja?

Reply:
Not yet. Let me try...

Reply:
In that case, the error is the same as initially https://github.com/AnacondaRecipes/optree-feedstock/pull/2#issuecomment-2713926508